Terasology/Lost
# WIP : Lost Remake

Progress : 
- Add level spawns at biome centre
- Add a progress tracking component which checks if well is found
- Remove unwanted Lost files and modules

Workarounds used :
- The world object is statically stored in the Lost world generator class (this should be fixed later)

Testing Instructions :
1. Select Lost as a gameplay template.
2. Check the in-game console for coordinates of new levels and the biome entered for faster testing.
